frontend/src/components/orgFacilities/ManageFacilities.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <v-container class="pa-0 text-body-1" fluid>
+    <v-row no-gutters>
+      <v-col>
+        <p>View and update your facilities.</p>
```

Comment:
Current wireframe has **View and manage the facilities of your organization.**

Reply:
Updated